internal/controller/shared.go

```diff
@@ -44,13 +44,14 @@ const (
 )
 
 // makeChildResourceFromRolloutAndUpdateSpecHash makes a new kubernetes.GenericObject based on the given rolloutObj.
-// It returns the child resource object ready to be created and an operation to be performed with the returned object.
+// It returns the child resource object ready to be created, an operation to be performed with the returned object,
+// and the calculated hash of the raw spec of the rollout to be set in the rollout annotations after succesful deployment.
 // The operations are defined by the RolloutChildOperation constants.
 func makeChildResourceFromRolloutAndUpdateSpecHash(
```

Comment:
We no longer update the Hash in this function, right? If so, the name should be updated accordingly.